app/assets/stylesheets/active_skin.sass

```diff
@@ -31,54 +31,65 @@ body.active_admin
     padding-left: 130px
     color: $skinTextColor
     text-indent: -999999px
+  .header-item
+    top: 0
   a, a:link
     color: $skinTextColor
   ul.tabs
     padding: 0
     font-size: 0
-  ul.tabs > li:hover > a, ul.tabs > li.current > a
-    color: $skinTextActiveColor
-    background: $skinActiveColor
-    @include rounded(0)
-    margin: -10px 0
-  ul.tabs > li 
-    font-size: 12px
-    margin: 0
-    a
-      padding: 13px 20px
+    & > li.menu_item
+      font-size: 12px
       margin: 0
-  ul.tabs > li.has_nested a
-    background-position: 93% 50%
```

Comment:
This rule and the other two instances are removed. The default rule from ActiveAdmin is more sensible to me in this case:

```css
background-position: calc(100% - 7px) 50%;
```

This way, the arrow is expected to be exactly 7 pixels from the right, which works well for menu with a long label.